src/Protocol/Email.php

```diff
@@ -5,7 +5,6 @@
 namespace Friendica\Protocol;
 
 require_once 'include/html2plain.php';
-require_once 'include/msgclean.php';
 require_once 'include/quoteconvert.php';
```

Comment:
Didn't you remove the quotconvert.php file?

Reply:
Ops

Reply:
Done